src/components/__tests__/css-prop.test.js

```diff
@@ -0,0 +1,16 @@
+import styled from '@emotion/styled'
+import serializer from 'jest-emotion'
+import React from 'react'
+import renderer from 'react-test-renderer'
+
+expect.addSnapshotSerializer(serializer)
+
+test('renders with correct styles', () => {
+  const H1 = styled.h1`
+    float: left;
+  `
+
+  const tree = renderer.create(<H1>hello world</H1>).toJSON()
```

Comment:
You can see how I tested other components using `react-testing-library`'s `render` function.
Please follow that as an example.
Thanks!